### PR TITLE
docs: Add sample script - user zones update

### PR DESCRIPTION
**Pull request by arjankowski**

### CODE REVIEW

File: examples/Mass Update User Zones/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Copyright (c) Microsoft Corporation.
+
+https://aka.ms/powershell
+Type 'help' to get help.
+
+PS /Users/mikechen>
+```
+
+4. Setting up the CLI requires creating a Box application that uses server authentication with JWT. See the [Box CLI Using JWT Authentication](https://developer.box.com/guides/tooling/cli/jwt-cli/) page for instructions. 
+    * **The Box application must have the following application access level enabled** in the configuration page: `App + Enterprise Access`
```
Comment (mgrytsai) [inline examples/Mass Update User Zones/README.md:33]: for readability suggest to bold only access level, application scopes, advanced features

Comment (arjankowski) [reply examples/Mass Update User Zones/README.md]: Done

**Review by mgrytsai — APPROVED** [review]
Looks great!

File: examples/Mass Update User Zones/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	|sally@company.com | JapanSingapore |
+
+2. Update the [adminEmail](/examples/Mass%20Update%20User%20Zones/Mass_Update_User_Zones.ps1#L21) to the admin or co-admin login email address of the account that will be used to make zone assignments.  If it is not specified in the script, you will be prompted for it.
+3. Optional: To run the script in simulate mode, set the `simulate` boolean flag when running the script:
+`./Mass_Update_User_Zones.ps1 -simulate`
+
+## Variables ##
+* `ZonesTable`: This is the logic that defines the zones. In this hashtable, the keys are the "friendly name" for the zone (this is the value used in the input .csv) and the corresponding value is the global ID for that zone. The customer should consult their BC or CSM contact to get the IDs corresponding to the zones enabled in their enterprise.
+
+	```powershell
```
Comment (mgrytsai) [inline examples/Mass Update User Zones/README.md:57]: Please refrain from listing the provider name with policies

**Pull request merged**